CacheWarmer/PersistentCollectionCacheWarmer.php

```diff
@@ -49,12 +49,8 @@ public function isOptional()
         return false;
     }
 
-    /**
-     * @param string $cacheDir
-     *
-     * @return string[]
-     */
-    public function warmUp($cacheDir)
+    /** @return string[] */
+    public function warmUp(string $cacheDir)
```

Comment:
[`WarmableInterface::warmUp()` already has `string` in 5.4](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/HttpKernel/CacheWarmer/WarmableInterface.php#L26)